### ISSUE TITLE
Add build instructions

### ISSUE THREAD

**Issue by malban** [hatchbed/log_view#14]
Refs #13

**Comment by marmik18** [comment]
Can we get the binary version for humble at `ros-humble-log-view`? I can help, if required.

**Comment by malban** [comment]
Yes,  I will take a look at this.   I created a new issue: https://github.com/hatchbed/log_view/issues/18
